### PR TITLE
Compiling hybrid application with gradle instead of using cordova build

### PR DESCRIPTION
**Pull request by wmathurin**

### CODE REVIEW

File: node/test_force.js [hunk 1]
```diff
@@ -257,8 +257,15 @@ function createCompileApp(tmpDir, appType, os) {
...
     else {
-        // IOS or Android - Hybrid
-        runProcessCatchError('cordova build', 'COMPILING ' + target, appDir);    
+        if (os == OS.ios) {
+            // IOS - Native
+            runProcessCatchError('cordova build', 'COMPILING ' + target, appDir);    
+        }
+        else {
+            // Android - Native
+            var gradle = isWindows() ? '.\\gradlew.bat' : './gradlew';
```
Comment (bhariharan) [inline node/test_force.js]: Do we need to explicitly run the postinstall script here?

**Comment by bhariharan** [comment]
Looks good otherwise.

File: node/test_force.js (hunk 1 above)
Comment (wmathurin) [inline node/test_force.js]: I'm running forcedroid first. This is just to see if the generated app compiles.

**Comment by khawkins** [comment]
:+1:

**Pull request merged**